src/App.js

```diff
@@ -1,9 +1,37 @@
-import "./App.css";
-
-function App() {
-  return (
-    <div>App.js</div>
-  );
-}
-
-export default App;
+import React from 'react'
+import { Routes, Route } from 'react-router-dom'
+import './App.css'
+import Header from './components/Header/Header'
+import SideBar from './components/SideBar/SideBar'
+import SignupPage from './pages/AuthPage/SignupPage'
+import LoginPage from './pages/AuthPage/LoginPage'
+import Homepage from './pages/Homepage/Homepage'
+import PlaylistPage from './pages/PlaylistPage/PlaylistPage'
+import LikedPage from './pages/LikedPage/LikedPage'
+import WatchLaterPage from './pages/WatchLaterPage/WatchLaterPage'
+import HistoryPage from './pages/HistoryPage/HistoryPage'
+import Mockman from 'mockman-js'
+const App = () => {
+  return (
+    <div className="main-container">
+      <Header />
```

Comment:
Suggestion: Don't put header and sidebar on all the routes.
If someone lands on the route that is not listed here also will be able to see the header an